python_modules/libraries/dagster-looker/dagster_looker/api/resource.py

```diff
@@ -126,16 +140,52 @@ def _build_defs_from_looker_instance_data(
             )
             for lookml_explore in looker_instance_data.explores_by_id.values()
         ]
-        views = [
+        dashboards = [
             dagster_looker_translator.get_asset_spec(
                 LookerStructureData(
                     structure_type=LookerStructureType.DASHBOARD, data=looker_dashboard
                 )
             )
             for looker_dashboard in looker_instance_data.dashboards_by_id.values()
         ]
+        dashboards_postprocessed = self._build_dashboard_notif_defs(
+            scheduled_plans_by_dashboard_id,
+            list(looker_instance_data.dashboards_by_id.values()),
+            dashboards,
```

Comment:
Maybe we should rename the objects above by `*_specs`? I got confused looking at the signature of `_build_dashboard_notif_defs` and the objects we are passing here.
```suggestion
            dashboards_specs,
```